stable/mlrun/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: mlrun
 version: 0.10.3
```

```suggestion
version: 0.10.4
```
need to bump the chart version as well